src/data/builder-tools.js

```diff
@@ -802,6 +802,13 @@ export const Showcases = [
     getstarted: "/docs/operate-a-stake-pool/frankenwallet",
     tags: ["operatortool"],
   },
+  { 
+    title: "cnft", 
+    description: "A library for validating metadata for the 721 metadatum property. Complies with community standards", 
+    preview: require("./builder-tools/cnft9000.png"), 
```

Comment:
This filename (`cnft9000.png`) is a better one than the actual filename currently in this PR (`ada9000.png`) because it's specific to the project, rather than the author.  Note the filenames must match in any case 🤓